segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/security/TLSHelper.java

```diff
@@ -47,7 +46,6 @@
     public static SslContext newServerSslContext(String pathToCertificateFile, String pathToServerKeyFile, String tlsProtocolVersion) {
         Exceptions.checkNotNullOrEmpty(pathToCertificateFile, "pathToCertificateFile");
         Exceptions.checkNotNullOrEmpty(pathToServerKeyFile, "pathToServerKeyFile");
```

Comment:
We will still need to check the argument.  But the checking can be done by checking `.tlsProtocols() != null` on the already parsed object.

Reply:
Added validation for tlsprotocols